ansible/roles_ocp_workloads/ocp4_workload_integreatly/tasks/post_workload.yml

```diff
@@ -106,6 +82,56 @@
     _instance_user: "{{ ocp4_workload_integreatly_user_base }}{{ _instance_count | int }}"
     _tenant_admin_secret_name: "{{ ocp4_workload_integreatly_user_base }}{{ _instance_count | int }}-admin-credentials"
 
+# Handle SSO IDP Setup
+
+- name: create sso idp
```

Comment:
Would it make sense to refactor that into its own role? Something like ocp4_workload_ingreatly_sso ? Or maybe even make it a generic workload that can set up SSO ?

Reply:
@wkulhanek The main reason I didn't pull this into a standalone role is because it depends heavily on the current RHMI 2.x SSO setup. Unlike the MinIO role, this really isn't very useful under other circumstances

Reply:
Makes sense. Just an idea. :-)